src/components/SearchBox/SearchBox.js

```diff
@@ -172,16 +172,18 @@ class SearchBox extends React.Component {
     const searchOptionValue = this.state.searchOption;
     const encoreBaseUrl = 'https://browse.nypl.org/iii/encore/search/';
     let catalogBaseUrl;
-    let appEnv = 'production';
-    if (!appEnv) {
-      catalogBaseUrl = '//www.nypl.org/search/';
-    } else if (appEnv === 'development') {
-      catalogBaseUrl = '//dev-www.nypl.org/search/';
-    } else if (appEnv === 'qa') {
-      catalogBaseUrl = '//qa-www.nypl.org/search/';
-    } else {
+    try {
+      if (appEnv === 'development') {
+        catalogBaseUrl = '//dev-www.nypl.org/search/';
+      } else if (appEnv === 'qa') {
+        catalogBaseUrl = '//qa-www.nypl.org/search/';
+      } else {
+        catalogBaseUrl = '//www.nypl.org/search/';
+      };
+    }
+    catch(err) {
       catalogBaseUrl = '//www.nypl.org/search/';
```

Comment:
I think we can have a comment here to indicate that when the header is on old/new Drupal, appEnv will not be set so it will always get to the case of error.

Reply:
Thanks @ktp242 I added the comment here: https://github.com/NYPL/dgx-header-component/pull/69/files#diff-7c2753b6b4dc6728a510c14dca7f1173R185

Reply:
👍 1+

Reply:
I would have said that that case is for the header markup and static assets import, and that one example is the Drupal import. Just to make it more generic but not a big deal.

Reply:
@EdwinGuzman I added a small PR to include the details you mentioned: https://github.com/NYPL/dgx-header-component/pull/70

I figure we can merge that into development and not bother giving the header component a new version until someone else makes a more substantial change in the future.

Reply:
That's a good plan. No need for a new version for something this small.